rfc/src/rfcs/0012-list.md

```diff
@@ -132,6 +152,30 @@ For example:
 }
 ```
 
+#### Output Schema
+The formal output schema is:
+```
+Kani Version: float
```

Comment:
This is not a float though. It's a String that represents semantic versioning (https://semver.org/#backusnaur-form-grammar-for-valid-semver-versions)